ubpf_plugin/CMakeLists.txt

```diff
@@ -34,18 +34,13 @@ else()
     message(STATUS "Using custom bpf_conformance_runner: ${BPF_CONFORMANCE_RUNNER}")
 endif()
 
-if(CMAKE_SYSTEM_PROCESSOR STREQUAL aarch64)
```

Comment:
Why did you remove this line? This is needed to run aarch64 tests under qemu.

Reply:
As mentioned in direct communication, removing that line was an absolute typo. I had removed it to enable executing tests natively on an arm-based machine and committed it. I am terribly sorry!